scripts/bootstrap/compile.sh

```diff
@@ -207,10 +207,24 @@ if [ -z "${BAZEL_SKIP_JAVA_COMPILATION}" ]; then
     cp src/main/java/$i ${OUTPUT_DIR}/classes/$i
   done
 
+  # Create the bazel_tools repository.
+  BAZEL_TOOLS_REPO=${OUTPUT_DIR}/embedded_tools
+  mkdir -p ${BAZEL_TOOLS_REPO}
+  cat <<EOF >${BAZEL_TOOLS_REPO}/WORKSPACE
+worksapce(name = 'bazel_tools')
+EOF
+  ln -s `pwd`/src ${BAZEL_TOOLS_REPO}/src
```

Comment:
`ln` makes copies on Windows.
Consider special-casing Windows and using `cmd.exe /C 'mklink /J "linkname" "linktarget"'` instead (with this quoting style).

Reply:
Actually, given the long path failure you saw, please do use mklink. That works with long paths.

Reply:
Thanks, trying this approach now.